app/controllers/healthcheck_controller.rb

```diff
@@ -26,4 +21,14 @@ def sendgrid_alive?
   def messagelabs_alive?
     SendgridHelper.smtp_alive?(*PrisonMailer.smtp_settings.values_at(:address, :port))
```

Comment:
Not related to @threedaymonk's work in this PR.. I don't like the fact that what should probably be `MailerHelper` or similar is called `SendgridHelper`.

Knowing that the `PrisonMailer` isn't using Sendgrid is a little confusing. :laughing: